framework/include/materials/MaterialPropertyInterface.h

```diff
@@ -310,6 +310,7 @@ class MaterialPropertyInterface
 
   /**
    * Retrieve the property deduced from the name \p name for the specified \p material_data
+   * that can be defined on element, face and neighboring face.
```

Comment:
```suggestion
   * that can be defined on element, neighbor and face.
```

Reply:
`neighbor` could mean neighboring element, so I did that on purpose to say neighboring face.

Reply:
it doesnt do neighboring face? It does neighbor element

Reply:
or like neighbor qp probably

Reply:
Not exactly, it is the same side but on the neighboring side. Basically for a side, we have `+` and `-` quantities for face and neighboring face respectively.

Reply:
it s retrieved on the face qps, but it s only defined on the neighbor elements right? It doesnt have to be, but TwoMatPropInterface is designed for that purposed

Reply:
Yes.

Reply:
I think @YaqiWang's phrasing using the "neighbor face" is the most accurate. I would not want just generic "neighbor" language as that can lead users to think volumetric neighbor